src/components/ebay-snackbar-dialog/test/mock/mock-basic.marko

```diff
@@ -0,0 +1,17 @@
+<ebay-snackbar-dialog open=state.open on-close('closeSnackbar')>
```

Comment:
Is this needed anymore?

Reply:
yes, if they want to re-open the snackbar later. 

I will add an example with only the open state for clarity.

Reply:
This is in mocks, not in examples; I don't think this is needed.